src/pages/BrowserPage.qml

```diff
@@ -151,7 +151,7 @@ Page {
         fullscreenHeight: portrait ? Screen.height : Screen.width
         portrait: browserPage.isPortrait
         maxLiveTabCount: 3
-        toolbarHeight: overlay.toolBar.rowHeight
+        toolbarHeight: Math.min(overlay.toolBar.rowHeight, webView.fullscreenHeight - overlay.yPosition)
```

Comment:
Why this?

Reply:
When the SecondaryBar is open, user is focused on it. At this time, the user is not working with the page and I think there is no need to compress it to keep important elements in scope.